static/app/utils/issueTypeConfig/cronConfig.tsx

```diff
@@ -38,7 +38,8 @@ const cronConfig: IssueCategoryConfigMapping = {
       landingPage: Tab.DETAILS,
       events: {enabled: true},
       openPeriods: {enabled: false},
-      checkIns: {enabled: false},
+      uptimeChecks: {enabled: false},
+      cronChecks: {enabled: false},
```

Comment:
should this be `true`? or are we turning it on in a different pr?

Reply:
Actually I guess it doesn't matter since nothing reads it yet, 👀  but that line is in my next PR